docs/src/examples/components/Portal/Types/PortalExampleFocusTrapped.shorthand.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Button, Header, Portal } from '@stardust-ui/react'
+
+class PortalExamplePortal extends React.Component {
+  render() {
+    return (
+      <Portal
+        content={
+          <div
+            style={{
+              background: '#ddd',
+              position: 'fixed',
+              left: '40%',
+              top: '45%',
+              zIndex: 1000,
+              padding: '10px',
+            }}
+          >
+            <Header>This is a portal with focus trap!</Header>
+            <p tabIndex={0}>Portals have tons of great callback functions to hook into.</p>
+            <p tabIndex={0}>To close, simply click the close button or click away</p>
+            <Button size="small" content="Button" />
+            <Button size="small" content="Another button" />
+          </div>
+        }
+        trigger={<Button content={'Toggle portal'} />}
+        trapFocus={true}
+        focusTrapZoneProps={{
+          isClickableOutsideFocusTrap: true,
```

Comment:
If this is set to `false` any click outside resets focus to the first element in the portal. Is that ok? @kolaps33

Reply:
yes, I know, there is no point to set it explicitly. Just wanted to show an example of how focus trap zone props are passed to the component. I would then better make an example with ```false``` value